postreise/analyze/generation/emissions.py

```diff
@@ -11,16 +11,20 @@
 
 
 def generate_emissions_stats(scenario, pollutant="carbon", method="simple"):
-    """Generate emissions statistics from the input generation data. Method
-    descriptions: 'simple' uses a fixed ratio of CO2 to MWh, 'always-on' uses
-    generator heat-rate curves including non-zero intercepts, 'decommit' uses
-    generator heat-rate curves but de-commits generators if they are off
-    (detected by pg < 1 MW).
+    """Calculate hourly emissions for each generator.
 
     :param powersimdata.scenario.scenario.Scenario scenario: scenario instance.
     :param str pollutant: pollutant to analyze.
     :param str method: selected method to handle no-load fuel consumption.
-    :return: (*pandas.DataFrame*) -- emissions data frame.
+    :return: (*pandas.DataFrame*) -- emissions data frame. index: timestamps, column:
+        plant id.
```

Comment:
Maybe add that the values are the hourly emissions in tons?

Reply:
Done